lib/fluent/log.rb

```diff
@@ -421,13 +487,17 @@ def initialize
     def configure(conf)
       super
 
-      if level = conf['@log_level']
-        unless @log.is_a?(PluginLogger)
-          @log = PluginLogger.new($log.dup)
-        end
-        @log.level = level
-        @log.optional_header = "[#{self.class.name}#{plugin_id_configured? ? "(" + @id + ")" : ""}] "
+      if plugin_id_configured? || conf['@log_level']
+        @log = PluginLogger.new($log.dup) unless @log.is_a?(PluginLogger)
         @log.optional_attrs = {}
+
+        if level = conf['@log_level']
+          @log.level = level
+        end
+
+        if plugin_id_configured?
+          @log.optional_header = "[#{@id}] "
```

Comment:
Including plugin class name looks too verbose for me.
It's not useful for end users, and it's clear when `-v` specified via filename.